libsweep/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(sweep)
+
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Ubuntu 14.04 has CMake 2.8.12. and I'd like to make it as painless as possible at least going back to Trusty.